### ISSUE TITLE
Can't start modded Skyrim with Mod Organizer 2.0.7

### ISSUE THREAD

**Issue by sacrish** [TanninOne/modorganizer#938]
I had been using MO1 for Skyrim and MO2 for Fallout 4. Today I decided to migrate my Skyrim mods to MO2 so that I could avoid the trouble changing NMM link handler when switching games.

Here's what I did to migrate the mods:

> 1. Set up a new instance for Skyrim in MO2.
> 2. Copy & paste MO1 folders (downloads, mods, profiles, webcache, overwrite) to the new MO2 Skyrim instance directory.
> 3. Start MO2. I can see all my profiles, mods, INI tweaks. Everything seems good (**Except that the overwrite 'mod' is empty**).
> 4. Run Skyrim (with and without SKSE) from MO2. **TESV.exe runs for a few seconds, game window doesn't show up. Then TESV.exe shuts down, MO2 gets unlocked. No error message.**

I tried to find out what caused the issue and if I could fix it:

> 1. Create a vanilla profile in MO2, run Skyrim. **Game successfully starts**.
> 2. Switch back to my main profile, de-activate all .esp files in the right panel, run the game. **Game fails to start**.
> 3. Re-activate all .esp files. Disable all mods (in the left panel), run the game. **Game successfully starts**.
> 4. Enable a few mods in the left panel (especially the non-esp ones like Stable uGridsToLoad, PapyrusUtil, etc.). **Game successfully starts**. 
> 5. Through some tests, I found that a mod called UNP Ranger Armor (http://www.nexusmods.com/skyrim/mods/77215/?) seems to prevent Skyrim from launching. So I enabled every other mod except this one, and ran the game - **Game process successfully starts, but CTD after loading a save**. Since I have over 200 mods installed, I won't bother testing them one by one to find which one (or what combination) causes the CTD.
> 6. Copy & paste everything back to MO1, and run the game from MO1, **my heavily modded Skyrim runs flawlessly**.

My question is:

> 1. Did I follow the proper steps when migrating from MO1 to MO2? Did I miss anything that is necessary to make MO2 work with my Skyrim? 
> 2. What could possibly cause the issue? Is there any possible fix?
> 3. Should I just stick with MO1 for Skyrim until MO2 gets stable enough?

**Comment by Gernash** [comment]
everything works in MO2 for skyrim other then skyproc applications like Dual sheath redux,sum,varent creatures,etc
MO2 has issues in portable mode or if you change from the default directory or leave it in the default directory.

Personally I do not run skyrim in MO2 as so much I use does not work it I want to configure it
if you want to muck about with it then go here
http://wiki.step-project.com/User:Gernash/MODWiki#Converting_to_My_wiki_is_easy.....

It's for fallout 4 but you should get the idea.

**Comment by sacrish** [comment]
Thanks for the reply Gernash!

Nope I don't have any SkyProc mod installed.  The only ones which involve external exe are FNIS and Bodyslide.

I'm running MO2 in non-portable mode, the one with custom instances. I did change the base directory location so that my C: partition won't be filled up with mods. My MO2 configuration for Skyrim is more-or-less the same with Fallout 4, which works well. I Never had any issues with FO4 BTW.

So I'm still not sure what causes the issue & how to fix it.

And thanks for the guide for MO2. I find it helpful to my FO4 :)

**Comment by Gernash** [comment]
Finis and bodyslide both work with the workaround.

If you use fo4edit and make a patch does it appear in your overwrite?(not just the folder) the work around is so you can have stuff in a different drive (the page is a general workaround)

If you install only texture replacers then you will have no issues with MO2 CBBE,FO4Edit,LOOT or creation kit will all bug or fail to see installed mods.(is info is so you can test Using FO4Edit is the easiest or loot)

**Comment by Gernash** [comment]
OPPS you have to install FINIS and ALL it's extras in the same mod folder

**Comment by sacrish** [comment]
Okay I think I'm making some progress.

As far as I can see, 2 factors are leading to this issue:

1. At first I changed mods location by modifying the base directory option in MO2. It turns out that the overwrite mod won't work properly this way. So I followed your guide and redirected AppData\ModOrganizer to a custom location using mklink. Now the overwrite mod is working.

2. Somehow, MO2 refuses to load INI files in profiles folder. I made a copy of those INI files and put them into the vanilla Documents\My Games\Skyrim folder. **Now I'm able to start Skyrim and load a save**.

However, new symptoms came up. After a save was loaded, a warning message showed up (I think it came from SkyUI), saying SKSE scripts were overwritten and needed re-installation (which I did afterwards, no help). Besides, some other mainly SKSE-based mods also failed to load (for instance, Racemenu, that my character's face was scrambled; and my body was transparent).

**Comment by Gernash** [comment]
MO2 does not work with ini files as far as I can tell it use default ini mygames folder

SKSE has to be in the base skyrim install folder and run form there.(not in MO2 as people sometimes do it.)
Also shadowboost and enb if you use them.

Not much will work in skyrim till you sort out the SKSE, I'll check it out later and send solution/workaround if there is one(but I dont recall having a problem with skse)

**Comment by Gernash** [comment]
hmmm also MO2 nolonger does Archieve management as such, I recommend installing Tes5edit and running it with all mods selected as it may come up with a load order glitch in the plugin load order. The new Loot seems to be a bit wishy washy (at least for FO4)

**Comment by sacrish** [comment]
Yes I did put stuff like SKSE and ENB in Skyrim base directory. The exactly same setup of mods work well with MO1, but MO2.

**Comment by Gernash** [comment]
@sacrish 
I tried a basic load with 
alternate start
skyui
 finis
XP32 extended
race menu

it loaded and saved fine.

click the filter button in MO2 and press Contains SKSE Plugins, What's listed there?
